quattor/functions/strings.pan

```diff
@@ -0,0 +1,29 @@
+declaration template quattor/functions/strings;
+
+include { 'pan/functions' };
+
+#version comparison function (should not be here)
+function comparever = {
```

Comment:
@sartiran i just found https://github.com/quattor/template-library-core/blob/master/quattor/functions/package.pan#L4, is that something you could reuse?